LethalAvatars/PlayerAvatarAPI.cs

```diff
@@ -161,7 +161,8 @@ public static void ApplyNewAvatar(Avatar clonedAvatar, PlayerControllerB player,
         avatarDriver.lastServerItemGrab = player.serverItemHolder;
         Transform cameraTransform = metarig.Find("CameraContainer/MainCamera");
         avatarDriver.AnimationDone(clonedAvatar.GetComponent<Animator>(), cameraTransform, player.IsLocal());
-        registeredAvatars.Add(player, clonedAvatar);
+        avatarDriver.SetupAvatar();
+        registeredAvatars.Add(clonedAvatar);
```

Comment:
`registeredAvatars` is a `Dictionary<PlayerControllerB, Avatar>`, not `List<Avatar>`